edb/schema/expraliases.py

```diff
@@ -79,12 +82,15 @@ def command_for_ast_node(
         modaliases = cls._modaliases_from_ast(schema, astnode, context)
         ctx = AliasCommandContext(
             schema,
-            op=sd._dummy_object,
+            op=sd._dummy_object,  # type: ignore
```

Comment:
`_dummy_command`

Reply:
I tried `_dummy_command` earlier, then I hesitated since mypy wants an `ObjectCommand` and it's necessary to add a new `_dummy_object_command = ObjectCommand(classname="dummy")`. :+1: pushing it now